### DISCUSSION TITLE
DexiNed preprocessor for SargeZT's SoftEdge controlnet model

### DISCUSSION THREAD

**Discussion by eurotaku** [Mikubill/sd-webui-controlnet#2093] [Ideas]
Hi, first a big thanks for all your hard work to get sdxl controlnet support up and running so nicely on automatic1111 :)
huggingface user SargeZT has released a SoftEdge model for DexiNed masks and i wanted to kindly ask, if there may be any plans to implement that into the extension? Link: https://huggingface.co/SargeZT/controlnet-sd-xl-1.0-softedge-dexined

**Comment by sdbds** [comment]
R.I.P for SargeZT.
i will add it soon.

**Reply by CycloneRing** [reply]
Patrick Shanahan died? :|

**Reply by eurotaku** [reply]
yeah, sadly :(

**Comment by sdbds** [comment]
#2564 ADD it now

**Reply by eurotaku** [reply]
awesome, thx :)
